sdk/sdktypes/session.go

```diff
@@ -30,11 +34,13 @@ func (SessionTraits) Validate(m *SessionPB) error {
 }
 
 func (SessionTraits) StrictValidate(m *SessionPB) error {
+	if m.BuildId == "" && m.DeploymentId == "" {
```

Comment:
just one more comment on the same topic. I think maybe we should remove buildID from deployment. Then session will always have buildID but not always deployment. Model and concept of what is deployment should be redefined as well